client/cli/src/lib.rs

```diff
@@ -362,6 +362,9 @@ enum Command {
         /// JSON args that will be serialized and send with the request
         #[clap(default_value = "null")]
         arg: String,
```

Comment:
btw, I think `arg` is called `param(s)` in the rest of the code. Perhaps rename it to make it (a) more consistent and (b) more similar to the RPC call `method + param`. 

just a thought, feel free to ignore

Reply:
Great point. Maybe we should also rename `ApiRequestErased` to `RpcParamsErased` or something? It's a little weird that `request_with_strategy` takes `method` and `params` arguments, and `params` is a "request".